packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -262,10 +264,17 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     maxDominantSpeakers: MAX_AUDIO_DOMINANT_SPEAKERS
   });
 
+  /* @conditional-compile-remove(rooms) */
+  const permissions = _usePermissions();
```

Comment:
I have to add the hook in the VideoGallery component because `localParticipant` is a required prop. So there is no way for me prevent the PiPiP from the composite without a changing `localParticipant to an optional prop in the API.

Reply:
I would actually be for making the local participant an optional prop - no reason Contoso should be forced to provide it. But happy with the solution also

Reply:
Making the local participant optional will be considered when for the new Video Gallery feature in Q4.